libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/room/RustMatrixRoom.kt

```diff
@@ -188,12 +188,12 @@ class RustMatrixRoom(
         _membersStateFlow.value = MatrixRoomMembersState.Pending(prevRoomMembers = currentMembers)
         var rustMembers: List<RoomMember>? = null
         try {
-            rustMembers = innerRoom.members().use { membersIterator ->
+            rustMembers = innerRoom.membersBlocking().use { membersIterator ->
                 buildList {
                     while (true) {
                         // Loading the whole membersIterator as a stop-gap measure.
                         // We should probably implement some sort of paging in the future.
-                        yield()
+                        ensureActive()
```

Comment:
What difference does this make in practice?
(yield calls ensureActive internally)

Reply:
The yield is not working on this scenario, not sure why. Dont have time to investigate more.